packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -1782,10 +1782,19 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
       final String? aId = service.toId(a, group1);
       expect(service.toId(a, group2), equals(aId));
       expect(service.toId(a, group3), equals(aId));
-      await service.testExtension('disposeGroup', <String, String>{'objectGroup': group1});
```

Comment:
no changes were made to this test other than formatting changes and replacing the usages of service extension strings with the new enum values.